src/Microsoft.DiaSymReader/Utilities/AssemblyInfo.cs

```diff
@@ -1,12 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the License.txt file in the project root for more information.
-#if NET20
-
-[assembly: System.Runtime.Versioning.TargetFramework(".NETFramework,Version=v2.0")]
-
-#else
-
-[assembly: System.Security.AllowPartiallyTrustedCallers]
```

Comment:
@tmat want to make sure u see that I deleted this. It seems to serve no purpose and just cause issues. Want to make sure I'm not missing a reason for it being there.

Reply:
I don't recall why we had it. Seems unnecessary for netstandard2.0.